climada/engine/impact.py

```diff
@@ -45,7 +45,6 @@
 from rasterio.crs import CRS as rasterioCRS  # pylint: disable=no-name-in-module
 
 from climada.entity import Exposures, Tag
```

Comment:
```suggestion
from climada.entity import Exposures
from climada.util.tag import Tag
```
Should it not be like this?